crates/era-cheatcodes/tests/test.sh

```diff
@@ -58,4 +58,4 @@ command -v git &>/dev/null || {
 build_zkforge "${REPO_ROOT}"
 
 echo "Running tests..."
-RUST_LOG=debug "${BINARY_PATH}" test --use "./${SOLC}"
+RUST_LOG=warn "${BINARY_PATH}" test --use "./${SOLC}"
```

Comment:
Changing the default level to `warn` as the log output is too verbose currently, and we usually only want the reasons for failure to investigate.